apps/meteor/server/startup/migrations/v298.ts

```diff
@@ -0,0 +1,11 @@
+import { Settings } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 295,
```

Comment:
migration version is not matching the file name

Reply:
Oops, fixed :nerd_face: